cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/messages/DetailedMessageBatchEntity.java

```diff
@@ -80,7 +84,27 @@ public DetailedMessageBatchEntity(StoredMessageBatch batch, UUID instanceId) thr
 		this.setMessageCount(batch.getMessageCount());
 		this.setLastMessageIndex(batch.getLastMessage().getIndex());
 	}
-	
+
+	public StoredMessageBatch toStoredMessageBatch(StoredMessageBatch messageBatch)
```

Comment:
Add a comment why you need the input parameter. This is because it should be created by CradleObjectsFactory, if I'm not mistaken.